v2/controllers/marketplace/clusterregistration_controller.go

```diff
@@ -53,6 +53,13 @@ type ClusterRegistrationReconciler struct {
 	Log    logr.Logger
 }
 
+// +kubebuilder:rbac:groups="",resources=secret,verbs=get;list;watch
```

Comment:
is read access to any Secret really required? perhaps can be restricted to specific namespace?